### PR TITLE
AB2-21: add MyOffrz

### PR DESCRIPTION
**Pull request by moaz-cliqz**
@spacifici please review it,
I have a suggestion to refactor "OnLoadFinish()" to make RecyclerView instead of adding childViews to the "offersContainer".
what do you think ?

### CODE REVIEW

File: mozilla-release/mobile/android/app/build.gradle [hunk 1]
```diff
@@ -303,6 +303,14 @@ dependencies {
 
     // Including the Robotium JAR directly can cause issues with dexing.
     androidTestCompile 'com.jayway.android.robotium:robotium-solo:5.5.4'
+
+    /* Cliqz start */
+    // add butter knife for view binding
+    compile ('com.jakewharton:butterknife:8.8.1') {
```
Comment (spacifici) [inline mozilla-release/mobile/android/app/build.gradle]: We have already a ton of dependencies, I think we have to wave butterknife and do it manually

File: mozilla-release/mobile/android/app/src/main/res/layout/home_myoffrz_panel.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (spacifici) [inline mozilla-release/mobile/android/app/src/main/res/layout/home_myoffrz_panel.xml]: Add the following as first line
```xml
<!-- Copyright © Cliqz 2018 -->
```
Just to mark the files we add as belonging to Cliqz

File: mozilla-release/mobile/android/app/src/photon/res/values/colors.xml [hunk 3]
```diff
@@ -271,12 +271,21 @@
     <color name="pwa_confirm_btn_normal">#E60A84FF</color>
 
     <!-- Cliqz start -->
+    <!-- General -->
+    <color name="accent_color">#00aef0</color>
```
Comment (spacifici) [inline mozilla-release/mobile/android/app/src/photon/res/values/colors.xml]: `accent_color` is not a very good name (does not give us any context), it comes from Lightning and we should rename it consistenlty

File: mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomeConfig.java [hunk 4]
```diff
@@ -1601,7 +1606,8 @@ public void remove() {
         public void setOnReloadListener(OnReloadListener listener);
     }
 
-    // UUIDs used to create PanelConfigs for default built-in panels. These are
+    //
```
Comment (spacifici) [inline mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomeConfig.java]: What's happening here?

Comment (moaz-cliqz) [reply mozilla-release/mobile/android/base/java/org/mozilla/gecko/home/HomeConfig.java]: adding UUIDS, generated random, I don't know how, they stored it fixed in the code.
but there is some others parts in the code may related to this, like "UUIDUtil.java" but they don't use it.
so I generate a key for MYOFFRZ tab like it.
 with 8-4-4-4-12 digits.

File: mozilla-release/mobile/android/app/build.gradle (hunk 1 above)
Comment (moaz-cliqz) [reply mozilla-release/mobile/android/app/build.gradle]: I think it's better to keep it. our layout is fully depending on it.
and it is much easier while coding.

File: mozilla-release/mobile/android/app/src/main/res/layout/home_myoffrz_panel.xml (hunk 2 above)
Comment (moaz-cliqz) [reply mozilla-release/mobile/android/app/src/main/res/layout/home_myoffrz_panel.xml]: done, also for other xml files

File: mozilla-release/mobile/android/app/build.gradle (hunk 1 above)
Comment (spacifici) [reply mozilla-release/mobile/android/app/build.gradle]: Let me rephrase this: do it manually
